analysis.go

```diff
@@ -84,6 +84,30 @@ func ListPackages(fileRoot, importRoot string) (PackageTree, error) {
 		if err != nil && err != filepath.SkipDir {
 			return err
 		}
+
+		// Read the destination of named symbolic link
+		// rules:
+		// 	1. All absolute symlinks are disqualified; if one is encountered, it should be skipped.
+		// 	2. Relative symlinks pointing to somewhere outside of the root (via ..) should also be skipped.
+		if !fi.IsDir() && fi.Mode()&os.ModeSymlink != 0 {
+			n := fi.Name()
```

Comment:
I don't think this is right - this just returns the basename of the path (the final element).